apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Full Site Editing
  * Description: Enhances your page creation workflow within the Block Editor.
- * Version: 0.16
+ * Version: 0.16.1
```

Comment:
We also need to update the constant a couple of lines below:

`define( 'PLUGIN_VERSION', '0.16' );`

Reply:
Done :)